snowshu/core/main.py

```diff
@@ -81,18 +81,22 @@ def source(filename):
 @click.option('--name',
               default=None,
               help="Overrides the replica name found in replica.yml")
+@click.option('--target',
```

Comment:
As mentioned before, we don't need this argument for the time being and it should be removed.